.github/workflows/ci.yml

```diff
@@ -124,6 +126,9 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v2
         with: {submodules: true}
+      - name: Change to older toolchain
+        if: matrix.os == 'macos'
+        run: sudo xcode-select -s "/Applications/Xcode_11.7.app"
```

Comment:
@mulimoen Just wondering why is this needed?

Reply:
The error message: `implicit declaration of function 'H5O__fsinfo_set_version' is invalid in C99 `.

The updated compiler is a bit better at enforcing `c99` standards, and `hdf5` placed the function definition below first use. This should be fixed in updated version of `hdf5`, we should update `hdf5-src` to pull the official fix.

Reply:
See #130 for how to fix this permanently